histomicstk/segmentation/label/trace_boundary.py

```diff
@@ -1,6 +1,6 @@
 import numpy as np
 import ctypes
-from .isbf import isbf
+from .trace_object_boundary import trace_object_boundary
 
 
 def trace_boundary(Mask, Connectivity=4, XStart=None, YStart=None,
```

Comment:
Also putting the loop over objects into cython or c++ will speed up things a lot.